README.md

```diff
@@ -1 +1,183 @@
-# Cryptocurrencies
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Cryptocurrencies] <a name="about-project"></a>
+
+
+
+**[Cryptocurrencies]** project is an app, that allows users to browse through a list of crypto coins fetched from an api and check out various details about the coins. It's built with React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+
+
+
+  <ul>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+
+
+
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Search_a_coin]**
+- **[Fetch_details]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ] [OPTIONAL] The link is not working, Kindly prove a live demo link.

Reply:
Fixed, thank you.